City-of-Pori/pori-kada
# PORI-242 Person reference field

To test:

1. drush fra -y && drush cc all
2. Go to any node that is page type and edit it.
3. There should be Person reference field. Add some people there.
4. Save the page and make sure the people are listed after body content.
